app/models/review.rb

```diff
@@ -1,9 +1,9 @@
 class Review < ApplicationRecord
   validates :rating, presence: true
-  validates :description, presence: true
+  validates :description
```

Comment:
looks like this validation was removed? If so, you should remove the line altogether